sdk/docs/storage/README.md

```diff
@@ -5,49 +5,67 @@ Azure Storage Blobs library (`az_storage_blobs`) provides abstractions, and help
 Use the Azure SDK Storage Blobs Library for Embedded C to work with Azure Storage Blobs.
 
 * Upload blobs
```

Comment:
This list just appears and the preamble sentence above it doesn't lead up to it. Reword?